recipes-bsp/u-boot/u-boot-iot2050.inc

```diff
@@ -64,6 +74,13 @@ dpkg_runbuild_prepend() {
     export TEE=/usr/lib/optee-os/iot2050/tee-pager_v2.bin
 }
 
+dpkg_runbuild_append_secureboot() {
+    if [ "${IOT2050_OTPCMD}" = "1" ]; then
+        dd if=${BUILDCHROOT_DIR}/usr/lib/secure-boot-otp-provisioning/otpcmd.bin \
+            of=${S}/${SPI_FLASH_IMG} bs=64K seek=108 count=1 conv=notrunc
```

Comment:
Why do we need `count=1` here? Is dd doing any padding this way? Do we need padding?

Also, this would be easier verifiable against the layout:
`dd bs=$((0x10000) seek=$((0x6c)) ...`